spec/system/provider_interface/candidate_pool/provider_views_candidate_pool_list_spec.rb

```diff
@@ -167,11 +167,16 @@ def when_i_filter_by_visa_sponsorship
 
   def then_i_expect_to_see_filtered_candidates(application_forms)
     candidates = page.all('.govuk-table__body .govuk-table__row td:first-child').map(&:text)
+    candidate_names = application_forms.map { |form| candidate_name(form) }
 
-    expect(candidates).to eq(application_forms.map(&:redacted_full_name))
+    expect(candidates).to eq(candidate_names)
```

Comment:
Just wondering if this might end up being flaky. Should we use`contain_exactly` instead of `eq`?